README.md

```diff
@@ -127,6 +127,15 @@ see it, do the following:
 
 TODO(rachelrosen): Document and demo how to add convai-checker webcomponent outside of the angular-cli build using the imported compiled javascript.
 
+### Publishing the webcomponent
+
+To publish a new version of the convia-checker webcomponent:
+1. Run `yarn run build:prod`
+2. Concatenate the JavaScript files with `cat build/static/runtime*.js build/static/polyfills*.js build/static/main*.js >build/static/perspective_authorship_widget.js`
```

Comment:
Cool! Might be worth even adding these commands to scripts in package.json so that we can run them with yarn or npm.

Reply:
Done - I've now combined the `yarn run build:prod` and `cat ...` steps into one script in package.json